client/src/components/ErrorMessage.js

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+
+export default function ErrorMessage({ text }) {
+  return (
+    <>
```

Comment:
I think you don't need the React Fragments here as the <Error> is the only return statement from the function